src/components/SanitizedHtml.js

```diff
@@ -28,6 +30,7 @@ export class SanitizedHtml extends Component {
         dangerouslySetInnerHTML={{ // eslint-disable-line react/no-danger
           __html: DOMPurify.sanitize(htmlString, htmlRules[ruleSet]),
         }}
+        {...props}
```

Comment:
Is this just to get the test id?

Reply:
Pretty much, but also seems like a nice thing for downstream consumes 🤷‍♂️